demos/sample_azure_iot_pnp/sample_azure_iot_pnp.h

```diff
@@ -0,0 +1,107 @@
+/* Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License. */
+
+#ifndef SAMPLE_AZURE_IOT_PNP_H
+#define SAMPLE_AZURE_IOT_PNP_H
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "azure_iot_hub_client_properties.h"

```

Comment:
>azure_iot_hub_client_properties [](http://example.com/codeflow?start=10&length=31)

how about including azure_iot_hub_client.h?

Reply:
Why should we include it? that API is not being used directly in this module.